spec/components/validators/css_color_validator_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CssColorValidator do
+  subject { described_class.new }
+
+  it "validates hex colors" do
+    expect(subject.valid_value?('#0')).to be_falsey
```

Comment:
Any reasons why you're not using `eq(false)` or `be false`?

We're rather be explicit about when we expect a method to return a boolean.